### ISSUE TITLE
Kubeflow Steering Committee Elections - Testimonial Phase - Yuan (Terry) Tang

### ISSUE THREAD

**Issue by akgraner** [kubeflow/community#679]
**Yuan (Terry) Tang**

**LinkedIn:** [https://www.linkedin.com/in/terrytangyuan](https://www.linkedin.com/in/terrytangyuan)

**Github:** [https://github.com/terrytangyuan](https://github.com/terrytangyuan)

Q: **Why do you think you would be a good candidate for the Kubeflow Steering Committee?**

Kubeflow enters the next critical phase by joining CNCF as an incubating project. As a current project lead and maintainer of Argo, I hope I can bring my experience with Argo (from incubating to graduated project within CNCF) to help with the transition of Kubeflow to CNCF and its future journey towards successful graduation.

In the meantime, my 6 years of involvement in the Kubeflow community as the tech lead and co-chair of WG Training gives me a holistic overview of our community and understanding of areas of improvements that we can address together. I will make my best decisions that benefit the community based on my first-hand experience in maintaining and advocating for the project.

Kubeflow is an umbrella of projects that offer great toolkits for ML applications in the cloud-native world with great community momentum and velocity. It’s both exciting and challenging as it involves so many areas in ML lifecycle and projects in the ecosystem and AI is gaining huge traction these days. I have held various leadership and maintainer positions in projects (e.g. Argo, TensorFlow, XGBoost) and organizations in the ecosystem (CNCF, JMLR, JOSS). I hope I can leverage my relationship with them to accelerate the growth of Kubeflow adoption.

Q: **How long have you been involved in the Kubeflow Community?  What projects have you been actively involved with?**

6 years. My work in the Kubeflow community focuses on the distributed training operator. I’ve been leading the development and maintenance of MPI Operator, XGBoost Operator, and MXNet Operator. I also lead the efforts to establish Kubeflow Common, which provides common and standardized APIs and libraries shared by other Kubeflow operator repositories. All these efforts are now merged into the unified training operator.

Besides the training operators, I’ve also been actively contributing to other components such as Kubeflow Pipelines and Katib through code, documentation, design discussions, etc.

Q: **How long have you been involved in open-source?**  

9 years.

Q:  **Is there anything else you would like the community to know about you that you believe would help eligible voters make there decision?**

I’d like to share some of my other non-code contributions/involvement related to Kubeflow over the years.

1. Authored the book Distributed Machine Learning Patterns that uses Kubeflow: https://bit.ly/2RKv8Zo 
2. Worked with other relevant projects and coordinating cross-projects collaborations/PRs such as Argo (project lead of Argo Workflows and maintainer of Argo CD), TensorFlow (maintainer), Apache MXNet (PMC), and XGBoost (PMC). The past communications and collaborations have helped integration of the Kubeflow training operator with various ML frameworks and ensures the smooth release of Kubeflow Pipelines (powered by Argo Workflows)
3. Advocated/spoke at various related events on Kubeflow (e.g. conferences like KubeCon, ODSC, ArgoCon, local meetups, and universities). A list of my public talks can be found here: https://github.com/terrytangyuan/public-talks 
4. Mentored students in Google Summer of Code in 2020 with a successful Kubeflow project.
5. Worked closely with users and partners to increase the adoption of Kubeflow (e.g. MPI Operator's industry adoption) and managing cross-company collaborations: https://github.com/kubeflow/mpi-operator/blob/master/ADOPTERS.md and https://medium.com/kubeflow/introduction-to-kubeflow-mpi-operator-and-industry-adoption-296d5f2e6edc 
6. Being program chair for KubeCon events related to Kubeflow, including Cloud-native AI Day and Data on Kubernetes Day.

Q: **Links to any external sites (projects, hobbies, etc) that you would like to share that would help people make a decision about.**

Summary of my open source contributions: https://github.com/sponsors/terrytangyuan 
Complete list of open source projects I’ve been involved in: https://terrytangyuan.github.io/projects/ 
List of public talks: https://github.com/terrytangyuan/public-talks

**Comment by jbottum** [comment]
Terry stands as a distinguished multi-year contributor to the Kubeflow community, leaving a lasting impact through critical contributions. As a dedicated code contributor, Terry’s expertise extends to comprehensive documentation efforts.   Terry's efforts include a recently published book on Distributed Machine Learning Patterns, which he authored and I recommend.   Displaying leadership qualities, Terry has taken on the role of Argo lead, showcasing a profound commitment to the success of that project and to Kubeflow.  Additionally, as a leader in both communities, Terry exemplifies organizational and coordination skills, while actively mentoring to nurture talent within the community. Terry has shared valuable insights as a speaker at a variety of conferences and venues, contributing significantly to the dissemination of knowledge. Notably, Terry's unwavering commitment to the Kubeflow community persists through changes in employment. Based in the US, Terry brings a crucial perspective to the collaborative and global efforts of the Kubeflow ecosystem. These qualifications make Terry a great candidate for the Kubeflow Steering Committee.

**Comment by andreyvelich** [comment]
I worked with Yuan on the Training Operator, MPI Operator, and Katib components for the last 5 years. He was actively involved in development of scalable distributed training capabilities in Kubeflow project. Yuan made significant contributions to the MPI Operator which allows many users adopt MPI Operator in production to train very large models and perform HPC tasks.

Yuan consistently contributes to the Kubeflow community growth by actively participating in conferences, writing books, blogs, and research papers, as well as integrating Kubeflow with other open source projects. For instance, we showcased integrations between Kubeflow Katib and Argo Workflows at the ArgoCon 2022.

As the Argo project lead, Yuan brings a wealth of experience in driving CNCF projects from incubation to graduation stages, which can significantly help with the Kubeflow project graduation in the future.
